pymc3/distributions/simulator.py

```diff
@@ -114,12 +114,12 @@ def random(self, point=None, size=None):
         -------
         array
         """
-        size = to_tuple(size)
-        params = draw_values([*self.params], point=point, size=size)
-        if len(size) == 0:
-            return self.function(*params)
-        else:
-            return np.array([self.function(*params) for _ in range(size[0])])
+        # size = to_tuple(size)
```

Comment:
Does this need refactoring, then?

Reply:
`Distribution.random` is gone forever; however, some of the code our `Distribution.random` implementations can be repurposed in a `RandomVariable.perform`, especially for `Distribution` that do not already have corresponding `RandomVariable` subclasses in Aesara.